CHANGELOG.md

```diff
@@ -1,47 +1,64 @@
-# 2.1.0
+# 2.0.0
+
+<!-- - Bugfix | Change | Feature | Documentation | Security -->
+
+## Data migration notes
 
-This release adds a logging collection to the database.
+In version 2.0.0, the ``project`` collection is partly filled by harvesting from
+external data source, and partly filled by means of users making edits in the
+admin interface. This means that version 2.0.0 of the Research Software
+Directory requires changes to the database. Below are the steps to migrate data
+from 1.2.0 to 2.0.0. Furthermore, the frontend now shows information for page
+maintainers, for which a new MongoDB collection ``logging`` is needed.
 
 ```
-docker-compose exec database mongo rsd
-db.createCollection("logging")
+$ docker-compose exec database mongo rsd
```

Comment:
Add link how to make a backup before starting the migration.

Reply:
Added a link to the relevant section in the README.